pages/developer-docs/installation/sunbird_backend.md

```diff
@@ -1,25 +1,21 @@
-## Introduction
-> This wiki contains following information.
-  1. Requirement
-  2. Steps to be followed for installing sunbird backend service on local machine
-  3. Testing Sunbird backend service
+This section elaborated the steps to be followed for installing sunbird backend service on local machine and testing Sunbird backend service
```

Comment:
@Basreena this file also (language edits needed) Since you only reviewed this.

Reply:
Reviewed again and merged,
This content will be live by 23/01/2018 EoD.
cc:
@Sunitashelar 
@manzarul